src/components/com_tjdashboard/administrator/models/dashboard.php

```diff
@@ -97,14 +97,43 @@ public function save($data)
 			return false;
 		}
 
+		$result = $dashboard->save();
+
 		// Store the data.
-		if (!$dashboard->save())
```

Comment:
@monikaatvowel  nothing special is done in this save method. Please check if you can remove this